File: src/main/kotlin/com/example/mirrors/exception/WebUserException.java
package com.example.mirrors.exception;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.lang.reflect.Array;
import java.lang.reflect.Method;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class WebUserException extends WebAppException {
    private Method method;
    private Array params;


}

File: src/main/kotlin/com/example/mirrors/exception/WebUserException.kt
package com.example.mirrors.exception

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import java.lang.reflect.Array
import java.lang.reflect.Method

@Data
@NoArgsConstructor
@AllArgsConstructor
class WebUserException(var method: Method, var params: Array?) : WebAppException(500,"UserException")